### DISCUSSION TITLE
How to parse freemarkup configuration yaml with dropwizard version 4.0.0

### DISCUSSION THREAD

**Discussion by harpreet86** [dropwizard/dropwizard#7187] [General]
Hi,
We were using _dropwizard-template-config_ library to parse our configuration yaml files written in free markup.
Now we migrated from dropwizard 2.1.4 to 4.0.0 and _dropwizard-template-config_ has become incompatible.

We don't want to update our configuration yaml files and there are no updates on _dropwizard-template-config_ since 2017.

Any suggestion to parse our freemarkup configuration yaml with dropwizard version 4.0.0?

**Comment by joschi** [comment]
Unfortunately I don't have another suggestion other than trying to contact the author of https://github.com/tkrille/dropwizard-template-config and ask them to update their third-party bundle to Dropwizard 3.0.x or 4.0.x.

This being said, Dropwizard supports reading configuration values from environment variables out-of-the-box: https://www.dropwizard.io/en/latest/manual/core/#environment-variables

Maybe that's already good enough for your use cases.
